src/index.js

```diff
@@ -179,6 +179,16 @@ class Bitswap {
     return this.engine.wantlistForPeer(peerId)
   }
 
+  /**
+   * Return ledger information for a given `peerId`
+   *
+   * @param {PeerId} peerId
+   * @returns {Object}
```

Comment:
It may also return `null`, hence it should be `@returns {?Object}`.

As you touch the code anyway, it would be cool to have a test which makes it return `null`.

Reply:
This is good feedback, no apology necessary.